shell/common/skia_event_tracer_impl.cc

```diff
@@ -183,22 +186,67 @@ class FlutterEventTracer : public SkEventTracer {
     trace_release_context(trace_context);
 
 #else   // defined(OS_FUCHSIA)
+    std::vector<const char*> arg_names;
+    std::vector<std::string> arg_values;
+    for (int i = 0; i < num_args; i++) {
+      arg_names.push_back(p_arg_names[i]);
+      const uint8_t arg_type = p_arg_types[i];
+      const uint64_t arg_value = p_arg_values[i];
+      switch (arg_type) {
+        case TRACE_VALUE_TYPE_BOOL: {
+          SkiaTraceValueUnion value = {.as_uint = p_arg_values[i]};
+          arg_values.push_back(std::to_string(value.as_bool));
+          break;
+        }
+        case TRACE_VALUE_TYPE_UINT:
+          arg_values.push_back(std::to_string(arg_value));
+          break;
+        case TRACE_VALUE_TYPE_INT:
+          arg_values.push_back(std::to_string(BitCast<int64_t>(arg_value)));
+          break;
+        case TRACE_VALUE_TYPE_DOUBLE:
+          arg_values.push_back(std::to_string(BitCast<double>(arg_value)));
+          break;
+        case TRACE_VALUE_TYPE_POINTER:
+          arg_values.push_back(std::to_string(BitCast<uintptr_t>(arg_value)));
```

Comment:
This is causing a static_assert to fire in skia.  Seems like it's not the right type.

Reply:
Done